event/listener.php

```diff
@@ -151,13 +151,8 @@ public function show_post_buttons($event)
 
 		if ($event['topic_data']['topic_first_post_id'] == $event['row']['post_id'])
 		{
-			$post_row = $event['post_row'];
-
-			$post_row['U_DELETE'] = false;
-			$post_row['U_QUOTE']  = false;
-			$post_row['U_WARN']   = false;
-
-			$event['post_row'] = $post_row;
+			$event->update_subarray('post_row', 'U_DELETE', false);
+			$event->update_subarray('post_row', 'U_WARN', false);
```

Comment:
Well, why wouldn’t we? It wasn’t before because of the Bot. But now, I think it should be for the same reasons any other post is quotable. Plus it’s been a request.